code challenges/14-all-methods/challenges-14.test.js

```diff
@@ -0,0 +1,229 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named toTitleCase that takes in an array of strings and returns an array of strings with the first character in upper case and the rest as is.
+
+For example, ['apple', 'banana', 'MacGyver'] returns ['Apple', 'Banana', 'MacGyver'].
+------------------------------------------------------------------------------------------------ */
+
+const toTitleCase = (arr) => arr.map(string => string.charAt(0).toUpperCase() + string.substring(1, string.length));
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named biggerThanLuke that, given the Star Wars data, below, returns the names of the characters whose mass is greater than Luke's.
+
+The names should be combined into a single string with each character name separated by a dash.
+
+For example, "Lando Calrisian - Boba Fett - Princess Amidala".
+------------------------------------------------------------------------------------------------ */
+
+let starWarsData = [{
+  name: 'Luke Skywalker',
+  height: '172',
+  mass: '77',
+  hair_color: 'blond',
+  skin_color: 'fair',
+  eye_color: 'blue',
+  birth_year: '19BBY',
+  gender: 'male',
+},
+{
+  name: 'C-3PO',
+  height: '167',
+  mass: '75',
+  hair_color: 'n/a',
+  skin_color: 'gold',
+  eye_color: 'yellow',
+  birth_year: '112BBY',
+  gender: 'n/a'
+},
+{
+  name: 'R2-D2',
+  height: '96',
+  mass: '32',
+  hair_color: 'n/a',
+  skin_color: 'white, blue',
+  eye_color: 'red',
+  birth_year: '33BBY',
+  gender: 'n/a'
+},
+{
+  name: 'Darth Vader',
+  height: '202',
+  mass: '136',
+  hair_color: 'none',
+  skin_color: 'white',
+  eye_color: 'yellow',
+  birth_year: '41.9BBY',
+  gender: 'male'
+},
+{
+  name: 'Leia Organa',
+  height: '150',
+  mass: '49',
+  hair_color: 'brown',
+  skin_color: 'light',
+  eye_color: 'brown',
+  birth_year: '19BBY',
+  gender: 'female'
+},
+{
+  name: 'Pex Kylar',
+  height: '180',
+  mass: '190',
+  hair_color: 'orange',
+  skin_color: 'brown',
+  eye_color: 'none',
+  birth_year: '27BBY',
+  gender: 'n/a'
+}];
+
+let biggerThanLuke = (arr) => arr.filter(character => character.mass > 77).map(character => character.name).join(' - ');
```

Comment:
Ideally you would not have hard coded Luke's mass and instead looped through the array to find it